CheckDrive.Api/CheckDrive.Application/DTOs/DispatcherReview/CreateDispatcherReviewDto.cs

```diff
@@ -4,6 +4,7 @@ namespace CheckDrive.Application.DTOs.DispatcherReview;
 
 public sealed record CreateDispatcherReviewDto(
     int CheckPointId,
+    int DriverId,
```

Comment:
CheckPointid should be enough, because CheckPoint.DoctorReview already contains DriverId we can take from there.